### ISSUE TITLE
Clarify what is a "testing party’s website" in the TCK License

### ISSUE THREAD

**Issue by bjhargrave** [EclipseFdn/EFSP#52]
The [Eclipse Foundation TCK license](https://www.eclipse.org/legal/tck.php) requires that the testing party must:

> make TCK test results showing full and complete satisfaction of all requirements of the TCK publicly available on the testing party’s website

It is not entirely clear what is acceptable for the _testing party’s website_.

Must this be a website under the testing party's commonly used domain name (_e.g._ `ibm.com` for IBM). Or it is acceptable that the website and/or its content merely be under the control of the testing party (_e.g._ a `github.io` or custom domain GitHub Pages site for an open source project)?

**Comment by mmilinkov** [comment]
I will, of course, defer to Wayne if he disagrees. But I think we would be happy with any website under the control of the party claiming compatibility. 

In this world of hosting services I don't see any value in being prescriptive about a domain name.

**Comment by bosschaert** [comment]
Would it be possible to clarify this in the TCK license? Maybe say something like:

> make TCK test results showing full and complete satisfaction of all requirements of the TCK publicly available on **_a_** website

instead?

**Comment by mmilinkov** [comment]
No, we are not going to change the text of the TCK License. That would require an exhaustive legal review and Board approval. Sorry, not gonna happen.

**Comment by bjhargrave** [comment]
Perhaps, in a future update,  the EFSP can define the term "testing party’s website" to bring more clarity. This would not require modifying the TCK license text.
